bin/goto

```diff
@@ -55,8 +56,7 @@ function _goto_main {
 
 
         # Special case 2  (goto <magicword>)
-        if [ "$#" -eq 1 ]; then
-
+        if [[ "$#" -eq 1 && ! "$1" =~ ^(list|help|add|rm|update|copy|subl|vscode|intelij|idea|show|cd|open)$ && ! "$1" =~ "--" && ! "$1" =~ "-"  ]]; then
```

Comment:
I found mysterious behaviour when intentionally crashing the `goto show`internals by `raise Exception('haha')`. lot's of commands ran twice. 
This was a first-aid attempt to break the recursion earlier.  

But this is hard to maintain - so I think a refactor will occur on these shell scripts at some point.